src/components/CommentList/CommentList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Comment, commentData } from '../Comment/Comment';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    <div className="comments">
+      {comments.map(comment => (
+        <Comment key={comment.id} comment={comment} />
+      ))}
+    </div>
+  </div>
+);
+
+CommentList.propTypes = {
+  comments: PropTypes.arrayOf(PropTypes.shape(commentData)).isRequired,
```

Comment:
Слишком много миллисекунд понадобилось чтобы понять что тут написано, давай переносить для удобства чтения.
```suggestion
  comments: PropTypes.arrayOf(
    PropTypes.shape(commentData)
  ).isRequired,
```